### ISSUE TITLE
Issues with Status Icon Counter

### ISSUE THREAD

**Issue by Flintwyrm** [schultzcole/FVTT-Combat-Tracker-Effects-Tooltips#1]
The mentioned Module allows for status Icons to have numbers tacked on to them. When Status Icon Counter is on it only show one Icon in the tracker instead of all the Icons on the actor. I am unsure if the problem lays in your code or theirs but I am sending this issue to both of you in hope that there can be a fix. thank you for this awesome module.

**Comment by DJ4ddi** [comment]
Reference: https://gitlab.com/woodentavern/status-icon-counters/-/issues/15

I don't see how adding a title attribute could break anything, but in my tests I couldn't reproduce the issue, so it's most likely a third party interfering. In any case, this issue is probably a no-op and can be closed.

**Comment by Flintwyrm** [comment]
Reference: https://gitlab.com/woodentavern/status-icon-counters/-/issues/15#notes
Sorry, I found the real culprit. Token Action HUD. I went one by one and added modules until the conditions disappeared from the combat tracker.

So if I have Status Icon Counter and Combat Tracker Effect Icon Tooltips ON alone they work, If I add Token Action HUD it does NOT work. BUT if I have Token Action HUD and Combat Tracker Effect Icon Tooltips ON without Status Icon Counter it works too?
